std/container/array.d

```diff
@@ -1014,13 +1021,46 @@ $(D r)
 
 @system unittest
 {
-    Array!int a;
+    struct Dumb { int x = 5; }
```

Comment:
Why not add a new unit test instead of changing the easier-to-understand shorter one?